Makefile

```diff
@@ -214,6 +226,31 @@ Switzerland North:cldrswitzerlandnorth,\
 Germany West Central:cldrgermanywestcentral,\
 Norway East:cldrnorwayeast
 endef
+
+ifndef AZURE_STORAGE_ACCOUNTS
+ifdef IMAGE_COPY_PHASE
+ifeq ($(IMAGE_COPY_PHASE),test)
+	AZURE_STORAGE_ACCOUNTS=$(AZURE_STORAGE_ACCOUNTS_PHASE1)
+endif
+ifeq ($(IMAGE_COPY_PHASE),prod)
+	AZURE_STORAGE_ACCOUNTS=$(AZURE_STORAGE_ACCOUNTS_PHASE2)
+endif
+endif
+ifndef AZURE_STORAGE_ACCOUNTS
+	AZURE_STORAGE_ACCOUNTS=$(AZURE_STORAGE_ACCOUNTS_PHASE1),$(AZURE_STORAGE_ACCOUNTS_PHASE2)
+endif
+endif
+
+
+# GCP AMI region definition
```

Comment:
AMI means Amazon Machine Images,so there is no such thing as GCP AMI

Edit: I see it was just moved so it is just a nice to have as a fix/improvement